app/views/exam_templates/_assign_errors.js.erb

```diff
@@ -0,0 +1,69 @@
+<script>
+  function rotate() {
+      PDFView.rotatePages(180);
+  }
+  PDFView.onLoadComplete = function() {
+      annotation_manager = new PdfAnnotationManager(PDFView, 'viewer');
+      annotation_manager.resetAngle();
+  };
+  var index = 0;
+  var files = <%= raw @error_files %>;
+
+  function getFile(filename) {
+      checkLastFile();
```

Comment:
A couple of style things for this file.

1. Use 2 spaces for indentation.
2. Use single-quotes, not double-quotes.
3. Use whitespace after `if`, after `)`, etc. (same comments as usual).